rtos/rtx/TARGET_CORTEX_M/cmsis_os.h

```diff
@@ -82,7 +82,12 @@
 
 #endif
 
+#if defined (MBED_CFG_DEBUG_OPTIONS_COVERAGE)
+/* GCOV works with stack size approx. 32K. */
+#define DEFAULT_STACK_SIZE         (WORDS_STACK_SIZE*4*8)
```

Comment:
I have two problems with this:

- if "gcov works with stack size approx. 32k", that should be stated explicitly in the definition of `DEFAULT_STACK_SIZE`, rather than depending on the definition of another macro (`WORDS_STACK_SIZE`) that can change unexpectedly. In other words: `#define DEFAULT_STACK_SIZE 32768`.
- that said, that's a pretty huge stack. Where is the "gcov works with stack size approx. 32k" coming from? I was not able to find this information with google.